builds/misc/mqtt-perf.yaml

```diff
@@ -0,0 +1,78 @@
+# MQTT Performance tests pipeline
+# Each platform separated into stages
+# Each stage consists of 
+# - setup job for test host (pull and start 'mqttd' container). See templates/mqtt-perf-setup.yaml
+# - setup job for publisher node and subscriber node (mqtt-benchmark tool writen in go). See templates/mqtt-perf-setup.yaml
+# - a set of test cases. See templates/mqtt-perf-test-case.yaml.
+
+# Pipeline requires custom agent pool (defined in $(pool.name)) 
+# with at least two separate agents:
+# - with 'mqtt-perf-tests-pub' demand - for publisher node
```

Comment:
What are the differences when set up agent as publisher compared to subscriber?

Reply:
no difference in agent setup, we just need to mark one of them as pub and another as sub, so they can perform corresponding roles in the test.

Reply:
would it be possible to pick up any agent from agent pool for publisher and subscriber? is it ok to use MS hosted agent?

Reply:
We have dedicated cluster set up and connected on a single switch, that's why we need to use specific agents. Ideally, I wanted to extract those agents into a separate pool, but I've been told to use existing one.

Reply:
That means we are going to set up 2 agents as a pair to run as publisher and subscriber explicitly.  And you are right, it is easy to use the existing agent pool.